tests/testcases.py

```diff
@@ -89,7 +89,10 @@ def setUp(self):
         write_default_config()
         save_config_value(constants.CONFIG_SETUP_KEY, True)
         save_config_value(constants.CONFIG_SPECS_REPO_KEY, 'github.com/gamechanger/dusty-example-specs')
-        save_config_value(constants.CONFIG_MAC_USERNAME_KEY, self.current_user)
+
+        self.tests_user = os.getenv('DUSTY_INTEGRATION_TESTS_USER', self.current_user)
```

Comment:
won't this get the value for the environment variable and use that, if it cannot find it then it uses self.current_user?